src/main/kotlin/uk/gov/dwp/dataworks/services/TaskDeploymentService.kt

```diff
@@ -39,11 +50,13 @@ class TaskDeploymentService {
     lateinit var taskRolePolicyDocument: Resource
     private lateinit var taskRolePolicyString: String
 
+    private lateinit var mapForIamParsing: Map<String, List<String>>
```

Comment:
Not sure what this is for but it's not used anywhere

Reply:
It was used in a previous version - I have removed it now.